intermediate_html_css/intermediate_css_concepts/positioning.md

```diff
@@ -69,3 +69,7 @@ The following questions are an opportunity to reflect on key topics in this less
 This section contains helpful links to related content. It isn't required, so consider it supplemental.
 
 - [Understand the CSS Position Property With Practical Examples](https://www.makeuseof.com/css-position-property-practical-examples/) provides some different CSS methods for positioning elements.
+ 
+- You can check out this helpful video resource on CSS positioning from [CSS positioning from Slaying the Dragon](https://www.youtube.com/watch?v=MxEtxo_AaZ4&t=2s) for clear explanations and practical examples.
```

Comment:
Just meant to move the opening `[` forward to extend the link text, no need to double up on the words here
```suggestion
- You can check out this helpful video resource on [CSS positioning from Slaying the Dragon](https://www.youtube.com/watch?v=MxEtxo_AaZ4&t=2s) for clear explanations and practical examples.
```